skill-tree /src/main/java/com/RDS/skilltree/Endorsement/EndorsementDTO.java

```diff
@@ -0,0 +1,36 @@
+package com.RDS.skilltree.Endorsement;
+
+import com.RDS.skilltree.EndorsementList.EndorsementListModel;
+import com.RDS.skilltree.Skill.SkillDTO;
+import com.RDS.skilltree.User.UserDTO;
+import com.RDS.skilltree.utils.TrackedProperties;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.*;
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class EndorsementDTO extends TrackedProperties {
+    private UUID id;
+    private UserDTO user;
+    private SkillDTO skill;
+    private EndorsementStatus status;
+    private List<EndorsementListModel> endorsersList;
```

Comment:
Please Note : The DTO for EndorsementListModel is not yet present in develop once it gets merged https://github.com/Real-Dev-Squad/skill-tree-backend/pull/41 I will transition EndorsementListModel to EndorsementListDTO